libgalois/src/PropertyGraph.cpp

```diff
@@ -346,13 +405,85 @@ katana::GraphTopology::Copy(const GraphTopology& that) noexcept {
 katana::Result<std::unique_ptr<katana::PropertyGraph>>
 katana::PropertyGraph::Make(
     std::unique_ptr<tsuba::RDGFile> rdg_file, tsuba::RDG&& rdg) {
+  katana::GraphTopology topo;
```

Comment:
This style of declaring all local variables at the top is bad. It's typically seen in C and Javascript programs. Sounds like a good idea but actually is bad for C++. Declare the variable where you need it and, more importantly, initialize it right away, unless there's a reason you can't.